libs/ucdao/src/lib/ucdao-confirm-modal.tsx

```diff
@@ -104,7 +116,8 @@ function DaoBalanceConfirmAmount({
   } else if (balanceNum === 0) {
     return (
       <>
-        <b>{tokensNum} LIQUID</b> token{tokensNum !== 1 ? 's' : ''}
+        <b>{tokensNum} LIQUID</b>{' '}
+        {t('token-count', { ns: 'uc-dao', count: tokensNum })}
```

Comment:
```suggestion
        <T
          keyName="token-count"
          ns="uc-dao"
          defaultValue="<b>{{tokensNum}} LIQUID</b> {{count}} points"
          params={{ tokensNum, count: tokensNum }}
          components={{ b: <b /> }}
        />
```

Reply:
1. why do we need to do like that?
2. T component doesn't have `components` prop  https://tolgee.io/js-sdk/integrations/react/api#t-component 
3. do we really need to translate LIQUID? Isn't it a special name like here for example:

lines 103-110
```tsx 
if (balanceNum !== 0 && tokensNum !== 0) {
    return (
      <>
        <b>{balanceNum} ISLM</b> {t('and', 'and')}
        <b>{tokensNum} LIQUID</b>
      </>
    );
  }
  ```

Reply:
Why do we need to do it this way?

The proposed structure aims to minimize string fragmentation in our translation code. By keeping phrases whole, we avoid separating prepositions or conjunctions, which makes the translation process smoother and reduces potential errors in context. It also enhances readability in code and ensures translators see more complete phrases, preserving contextual integrity across different languages. Additionally, this structure supports any language's grammatical structure without needing extensive reformatting or reordering.

T component and the components prop

You're correct that Tolgee’s <T> component doesn’t officially support a components prop as noted in the documentation. However, this approach might be a workaround using a different interpretation of the code or an alternative method. If this specific usage is not viable, we should stick to Tolgee’s recommended practices, potentially with custom elements or styles to manage translations effectively.

Translating the word “LIQUID”

"LIQUID" here seems like a branded or special term rather than one that requires translation, similar to “ISLM” in your example. Translating it could lead to inconsistent user experiences across regions where users expect certain terms to remain in English or another standardized form. Unless there's a strong reason (e.g., regulatory or brand guidelines) to translate "LIQUID," keeping it as-is would likely be more intuitive for users.

Hope this clarifies the reasoning. Let me know if further adjustments are needed!

Reply:
Thanks for clarifying the reasoning behind minimizing string fragmentation for translation consistency. It definitely makes sense to use a structure that adapts better to different languages.